roles/lvm/tasks/volume.yml

```diff
@@ -9,6 +9,16 @@
     - docker
     - bootstrap
 
+- name: update lvg ansible module
+  connection: local
+  run_once: yes
+  get_url:
+    url: https://raw.githubusercontent.com/ansible/ansible-modules-extras/devel/system/lvg.py
+    dest: "{{ playbook_dir }}/library/lvg.py"
+  tags:
+    - docker
+    - bootstrap
```

Comment:
In addition, if this is present and run in Packer, it will download the script to the machine and then we'll distribute it... which as you point out is not allowed with our licensing.